bin/poesie

```diff
@@ -17,6 +17,12 @@ rescue LoadError => e
   abort 'error: missing some dependencies for poesie. Be sure to install it using `gem install` or to run `bundle installl` to install dependencies - ' + e.message
 end
 
+module CurrentOS
+  UNKNOW = 0
+  ANDROID = 1
+  IOS = 2
+end
```

Comment:
I think instead we should have 2 constants:

```
ANDROID_FILTER_REGEX = /_android$/
IOS_FILTER_REGEX = /_android$/
```
And use those constants everywhere we need:

* Use the `IOS_FILTER_REGEX` constant directly in the implementation of `write_strings_file` and `write_stringsdict_file`
* Use the `ANDROID_FILTER_REGEX` constant directly in the implementation of `write_strings_xml`
* Pass the right constant as an additional argument when invoking `ContextFormatter::write_context_json` (instead of the CurrentOS argument that will now disappear)